go.mod

```diff
@@ -20,7 +20,7 @@ require (
 	github.com/yudai/hcl v0.0.0-20151013225006-5fa2393b3552 // indirect
 	github.com/yudai/umutex v0.0.0-20150817080136-18216d265c6b // indirect
 	go.uber.org/zap v1.16.0
-	golang.org/x/crypto v0.0.0-20210415154028-4f45737414dc
+	golang.org/x/term v0.0.0-20201126162022-7de9c90e9dd1
```

Comment:
Any reason (beyond go mod) an older version of this pkg is used?

Reply:
I just ran `hack/update-deps.sh`, and trusted it.

Reply:
I'm not sure, because it looks like there are more recent commits in https://github.com/golang/term

Reply:
Feel free to upgrade if it makes sense:

```console
$ go get -u golang.org/x/term
```

```patch
diff --git a/go.mod b/go.mod
index c1ec9aee..f174007a 100644
--- a/go.mod
+++ b/go.mod
@@ -20,7 +20,8 @@ require (
 	github.com/yudai/hcl v0.0.0-20151013225006-5fa2393b3552 // indirect
 	github.com/yudai/umutex v0.0.0-20150817080136-18216d265c6b // indirect
 	go.uber.org/zap v1.16.0
-	golang.org/x/term v0.0.0-20201126162022-7de9c90e9dd1
+	golang.org/x/sys v0.0.0-20210423185535-09eb48e85fd7 // indirect
+	golang.org/x/term v0.0.0-20210422114643-f5beecf764ed
 	gotest.tools v2.2.0+incompatible
 	k8s.io/api v0.19.7
 	k8s.io/apimachinery v0.19.7

```

Reply:
ping on this update ^